.github/workflows/higgs-shop-sample-app-release.yml

```diff
@@ -31,11 +31,65 @@ jobs:
                     echo "You can only run a release from the main branch, you are trying to run it from ${BRANCHNAME}"
                     exit 1
                   fi
-                  
+              
+    fetch-data-plans:
+        name: "Fetch Data Plan"
+        runs-on: ubuntu-latest
+        needs: confirm-main-branch
+        steps:
+          - name: Checkout
+            uses: actions/checkout@v2
+
+          - name: Use Node.js
+            uses: actions/setup-node@v1
+            with:
+              node-version: '16.x'
+
+          - name: Install mP CLI
+            run: npm install -g @mparticle/cli
+
+          - name: Fetch Data Plans with mp CLI
+            id: fetch-data-plans
+            env:
+              WORKSPACE_ID: ${{ secrets.HIGGS_SHOP_WORKSPACE_ID }}
+              CLIENT_ID: ${{ secrets.HIGGS_SHOP_CLIENT_ID }}
+              CLIENT_SECRET: ${{ secrets.HIGGS_SHOP_CLIENT_SECRET }}
+            run: |
+              mp planning:data-plan-versions:fetch --dataPlanId=higgs_shop_basic_data_plan --versionNumber=1 --outFile=dataplans/higgs_shop_basic_data_plan_1.json --workspaceId=$WORKSPACE_ID --clientId=$CLIENT_ID --clientSecret=$CLIENT_SECRET
+          
+          - name: View directory
+            run: pwd
+            
+          - name: View directory
+            run: ls -altr .
+            
+          - name: View directory
+            run: ls -altr dataplans
+            
+          - name: View Github Workspace
+            run: ls -altr ${{ github.workspace }}
```

Comment:
Is this for debugging purposes when you were building this script - if so should they be removed?  if not, i think you should change the `name` for each to more accurately reflect what it's doing

1. Print directory path
2. View all files 
3. View all files that start with `dataplans`

Reply:
Good catch. Removing them.